core/src/data/dataSource.h

```diff
@@ -69,8 +74,13 @@ class DataSource : public std::enable_shared_from_this<DataSource> {
     /* Generation ID of DataSource state (incremented for each update, e.g. on clearData()) */
     int64_t generation() const { return m_generation; }
 
+    int32_t minZoom() const { return m_minDisplayZoom; }
```

Comment:
Probably better to rename the accessor to `minDisplayZoom` to avoid any confusion.